src/helpers/charts/chart_common.ts

```diff
@@ -182,11 +182,6 @@ export function createDataSets(
           )
         );
       }
-    } else if (zone.left === zone.right && zone.top === zone.bottom) {
-      // A single cell. If it's only the title, the dataset is not added.
-      if (!dataSetsHaveTitle) {
-        dataSets.push(createDataSet(getters, dataSetSheetId, zone, undefined));
-      }
     } else {
       /* 1 row or 1 column */
```

Comment:
also restore the previous comment :wink: 
```suggestion
      /* 1 cell, 1 row or 1 column */
```